### PR TITLE
meow meow meow meow

### PR DESCRIPTION
**Pull request by mcovarr**
https://www.youtube.com/watch?v=9AWCk7HlG4o

Just an upgrade from 0.4.1 to 0.7.2 with a little bit of import cleanup here and there.

### CODE REVIEW

File: src/main/scala/centaur/test/Test.scala [hunk 1]
```diff
@@ -48,6 +47,9 @@ object Test {
         override def run: Try[A] = Try(x)
       }
     }
+
+    /** Call the default non-stack-safe but correct version of this method. */
+    override def tailRecM[A, B](a: A)(f: (A) => Test[Either[A, B]]): Test[B] = defaultTailRecM(a)(f)
```
Comment (ruchim) [inline src/main/scala/centaur/test/Test.scala:52]: ToL: This doesn't effect existing tests, right?

**Comment by geoffjentry** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/110/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Review by ruchim — APPROVED** [review]
Changes look good, meow-tastic :)

File: src/main/scala/centaur/test/Test.scala (hunk 1 above)
Comment (mcovarr) [reply src/main/scala/centaur/test/Test.scala:52]: This should not affect the existing tests.  Cats is just suggesting that the author of a `Monad` instance create a method for stack-safe traversal.  If one declines to do that (as I did here), one can just fall back on the non-stack-safe default implementation.

Comment (geoffjentry) [reply src/main/scala/centaur/test/Test.scala:52]: It's almost as if someone coded things in a way that would cause stack overflows 😨

**Comment by ruchim** [comment]
👍 woof

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/110/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**